src/test/java/org/jenkinsci/plugins/p4/PerforceScmTest.java

```diff
@@ -24,7 +24,7 @@ public void testConfigBasic() throws Exception {
 
 		String credential = "123";
 		Workspace workspace = new StaticWorkspaceImpl("none", false, "test.ws");
-		Populate populate = new AutoCleanImpl(true, true, false, null);
+		Populate populate = new AutoCleanImpl(true, true, false, false, null);
```

Comment:
I left all original booleans as is - just added a "false" parameter for the new parameter.